src/Microsoft.Docs.LearnValidation/Validators/ModuleValidator.cs

```diff
@@ -30,13 +30,13 @@ public override bool Validate(Dictionary<string, IValidateModel> fullItemsDict)
                 }
 
                 // module has child unit, but that unit has error when SDP validating
-                var childrenCantFind = module.Units.Where(u => !fullItemsDict.ContainsKey(u));
-                if (childrenCantFind.Any())
+                var childrenCantFind = module?.Units.Where(u => !fullItemsDict.ContainsKey(u));
```

Comment:
line 27 already assumes `module != null` - should we update that line as well?

Reply:
how about `continue` if `module == null`

Reply:
Thanks, will fix it.

Reply:
Sorry, I mixed up two PR, the issue possible null reference is tracked by another existing [PR](https://github.com/dotnet/docfx/pull/6918).